content/en/continuous_integration/tests/_index.md

```diff
@@ -112,20 +112,18 @@ All tags with the `test.configuration` prefix are used as configuration tags, in
 
 In order to filter using these configurations tags, [you must create facets for these tags][2].
 
-## Test suite level visibility
+## Test suite-level visibility
 
 In addition to tests, CI Visibility provides visibility over the whole testing phase of your project.
 
-### Compatibility
-
-Not every language supported by CI Visibility has support for test suite level visibility:
-
-* [Swift][3] has complete support since `dd-sdk-swift-testing>=2.1.0`.
-* [.NET][4] has complete support since `dd-trace-dotnet>2.16.0`.
-* [JavaScript][5] has limited support since `dd-trace-js>=3.3.0`.
-* [Java][6] has complete support since `dd-trace-java>=1.12.0`.
-* [JUnit report uploads][7] has complete support since `datadog-ci>=2.17.0`.
-* [Python][8] has complete support since `dd-trace-py>=1.14.0`
+{{< whatsnext desc="See the compatibility information for languages that support test suite-level visibility:" >}}
+    {{< nextlink href="continuous_integration/tests/setup/dotnet/?tab=testsuitelevelvisibility" >}}.NET{{< /nextlink >}}
+    {{< nextlink href="continuous_integration/tests/setup/javascript/?tab=testsuitelevelvisibility" >}}JavaScript{{< /nextlink >}}
+    {{< nextlink href="continuous_integration/tests/setup/swift/?tab=testsuitelevelvisibility" >}}Swift{{< /nextlink >}}
+    {{< nextlink href="continuous_integration/tests/junit_upload/?tab=testsuitelevelvisibility" >}}Uploading JUnit test report files to Datadog{{< /nextlink >}}
+    {{< nextlink href="continuous_integration/tests/setup/java/" >}}Java?{{< /nextlink >}}
+    {{< nextlink href="continuous_integration/tests/setup/python" >}}Python?{{< /nextlink >}}
```

Comment:
Need to verify if this information exists.

```suggestion
```

Reply:
Getting this info for Java is possible, although I'm not sure if it provides much value to the customer to know that TSLV for a given framework was supported in the tracer version X.
As we're encouraging the clients to always use the latest version of the tracer should we, perhaps, get rid of the TSLV-compatibility table completely?

Reply:
This is available for Python as well. 

I think it's okay to get rid of the versions since these have gone GA with TSLV - I don't think it's worth tracking capabilities for beta versions. 

I could see us tracking the features by version in the future - it gives the customer a better idea if they're on an older version of what they can expect by updating to the latest.